ledger-core/network/nds/msgdelivery/ctl_retry_state.go

```diff
@@ -0,0 +1,276 @@
+// Copyright 2020 Insolar Network Ltd.
+// All rights reserved.
+// This material is licensed under the Insolar License version 1.0,
+// available at https://github.com/insolar/assured-ledger/blob/master/LICENSE.md.
+
+package msgdelivery
+
+import (
+	"sync"
+	"time"
+
+	"github.com/insolar/assured-ledger/ledger-core/vanilla/throw"
+
+	"github.com/insolar/assured-ledger/ledger-core/network/nds/msgdelivery/retries"
+	"github.com/insolar/assured-ledger/ledger-core/network/nwapi"
+)
+
+type stateSender struct {
+	stages retries.StagedController
+
+	oob    chan rqShipment
```

Comment:
what oob stands for? outbound?

Reply:
https://en.wikipedia.org/wiki/Out-of-band_data